superset/assets/src/visualizations/PlaySlider.css

```diff
@@ -1,8 +1,7 @@
 .play-slider {
     position: absolute;
-    bottom: -16px;
     height: 20px;
-    width: 100%;
+    width: 90%;
```

Comment:
Try wrapping PlaySlider in `<div style="position: relative">`
It will be better to have the slider at full width rather than 90%

Reply:
+1